addons/binding/org.openhab.binding.pjlinkdevice/about.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
Replace with NOTICE file : https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088

Reply:
done